addons/mail/static/src/core/common/message_reaction_menu.js

```diff
@@ -59,4 +59,12 @@ export class MessageReactionMenu extends Component {
     getEmojiShortcode(reaction) {
         return this.store.emojiLoader.loaded?.emojiValueToShortcodes?.[reaction.content][0] ?? "?";
     }
+
+    get contentClass() {
+        const classes = ["o-mail-MessageReactionMenu", "h-50", "d-flex"];
+        if (this.store.useMobileView) {
+            classes.push("position-absolute", "bottom-0");
+        }
+        return classes.join(" ");
+    }
```

Comment:
Can use same technique as in ActionPanel.classNames, which uses object syntax for attClass and then turn into string with " ": https://github.com/odoo/odoo/blob/saas-18.4/addons/mail/static/src/discuss/core/common/action_panel.js#L22